instrumentation/rdkafka/lib/opentelemetry/instrumentation/rdkafka/instrumentation.rb

```diff
@@ -11,8 +11,11 @@ module Rdkafka
       class Instrumentation < OpenTelemetry::Instrumentation::Base
         MINIMUM_VERSION = Gem::Version.new('0.10.0')
 
+        GEM_VARIANTS = %w[rdkafka karafka-rdkafka]
+        
         compatible do
-          Gem.loaded_specs['rdkafka'].version >= MINIMUM_VERSION
+          variant = GEM_VARIANTS.find { |variant| !Gem.loaded_specs[variant].nil? }
```

Comment:
Is there a `VERSION` constant you can use here instead?

Reply:
You mean a direct reference via `Rdkafka::VERSION`?

Reply:
Yes please.